cmake/IncludeGrpc.cmake

```diff
@@ -71,6 +71,11 @@ if ("${GOOGLE_CLOUD_CPP_GRPC_PROVIDER}" STREQUAL "external")
 
     find_package(OpenSSL REQUIRED)
 
+    add_library(gRPC::address_sorting INTERFACE IMPORTED)
```

Comment:
Is this a new dependency of gRPC? What does it do?

Reply:
According to their release notes:

https://github.com/grpc/grpc/releases/tag/v1.11.0

sort addresses per [RFC-6724](https://tools.ietf.org/html/rfc6724), which is all about how to pick between the multiple addresses available in a host supporting IPv6 (and IPv6+IPv4 if it is a dual stack host).  I might be reading the abstract + intro wrong though.

Note that for our purposes this is just another small library inside the gRPC collection (along with `libgrpc++`, `libgrpc`, and `libgpr`).  The library code is fully included by gRPC (in their `third_party/` directory).  And their Makefiles / CMake / Bazel files compile and install the library.

Reply:
Would this be unnecessary (similarly to `-laddress_sorting` if https://github.com/grpc/grpc/issues/15166 were fixed? Or is there any other condition under which this block of code could be removed? If so, can you document that, please?